code/modules/research/designs/boards/machine_misc.dm

```diff
@@ -10,6 +10,17 @@
 	category = "Machine Boards"
 	build_path = /obj/item/weapon/circuitboard/recharge_station
 
+/datum/design/recharger
+	name = "Circuit Design (Recharger)"
+	desc = "Allows for the construction of circuit boards used to build Weapon Rechargers"
+	id="recharger"
+	req_tech = list(Tc_POWERSTORAGE = 2, Tc_COMBAT = 2)
+	build_type = IMPRINTER
+	materials = list(MAT_GLASS = 2000, SACID = 3)
```

Comment:
I don't really think that SACID is a real balancing factor for circuit imprinters because researchers will have a Chemistry Dispenser at the ready most times.
Besides, the recipe is also in the soldering iron, and that tool is hardcoded to consume only 4 units of acid per board no matter the type.
If you are certain that you want this change, tell me and I'll change it.

Reply:
No I was thinking about that as well after I wrote it.  Leave it as is and we'll look at it in in a different PR